alma/alma-8.x/common/install_mpis.sh

```diff
@@ -22,7 +22,7 @@ $COMMON_DIR/download_and_verify.sh $IMPI_2018_DOWNLOAD_URL "a1114b3eb4149c2f1089
 tar -xvf l_mpi_${IMPI_VERSION}.tgz
 cd l_mpi_${IMPI_VERSION}
 # Unsupported OS -
-# TODO: Update the silent.cfg file to proceed with installation
+# Update the silent.cfg file to proceed with installation
 sed -i -e 's/ACCEPT_EULA=decline/ACCEPT_EULA=accept/g' silent.cfg
 ./install.sh --silent ./silent.cfg
```

Comment:
Does this accept the option like `--accept` instead of we modify the silent.cfg?

Reply:
As per the official instructions at - https://registrationcenter-download.intel.com/akdlm/irc_nas/1718/INSTALL.html?lang=en&fileExt=.html
we can perform a silent installation by editing the config file. There is no additional --accept parameter